index.d.ts

```diff
@@ -153,6 +153,8 @@ interface BaseInputProps<T = HTMLInputElement> {
  * @property  { boolean } allowClear - [optional] show/hide clear fieldd control
  * @property  { boolean } showNote - [optional] show/hide note at suggestions list
  * @property  { boolean } silentQuery - [optional] initial query that not showed on input but determine  suggestion list that showed at first
+ * @property  { string } name - [optional] - property name of input (default: "daData")
+ * @property  { number } minimumCharacterThreshold - [optional] minimum number of characters entered after which a first request is sent (default: 3)
```

Comment:
I didn't notice this file when I was making some changes in this repo. 
Could you add one more prop that I added before (https://github.com/kzadurskakreditech/react-dadata-box/commit/33d51a9f11a0d310636fdfc9fce990b10f9ef559)
`autoFocus `

*@property { boolean } autoFocus - [optional] add autofocus to the field

Reply:
9e46878 ✅

Reply:
This whole file seems cool, we should think about adopting it in CCL